lightning/src/routing/scoring.rs

```diff
@@ -552,6 +555,12 @@ pub struct ProbabilisticScoringParameters {
 	/// When built with the `no-std` feature, time will never elapse. Therefore, the channel
 	/// liquidity knowledge will never decay except when the bounds cross.
 	pub liquidity_offset_half_life: Duration,
+
+	/// Whether an additional penalty should be calculated based on the payment amount in order to
+	/// provide higher penalties for large payments.
+	///
+	/// Default value: false
+	pub include_amount_penalty: bool,
```

Comment:
I'm not sure if we want to make this a switch instead of a second penalty - I think a second penalty more closely lines up with what fees are (a combo of % and absolute fees), which is the property we're going for here - ability to match penalties up to expected/reasonable fees.

Reply:
Updated as discussed offline. Please take a look at the `amount_penalty_multiplier_msat` docs. I tried to make them a bit more human understandable. Let me know if I hit the mark or if it can be fine tuned in any way.